renderdoc/driver/vulkan/wrappers/vk_device_funcs.cpp

```diff
@@ -1917,6 +1917,15 @@ bool WrappedVulkan::Serialise_vkCreateDevice(SerialiserType &ser, VkPhysicalDevi
       RDCLOG("Enabling VK_KHR_performance_query");
     }
 
+    bool multiviewperviewviewports = false;
+    if(supportedExtensions.find(VK_QCOM_MULTIVIEW_PER_VIEW_VIEWPORTS_EXTENSION_NAME) !=
+       supportedExtensions.end())
+    {
+      multiviewperviewviewports = true;
+      Extensions.push_back(VK_QCOM_MULTIVIEW_PER_VIEW_VIEWPORTS_EXTENSION_NAME);
+      RDCLOG("Enabling VK_QCOM_multiview_per_view_viewports");
+    }
```

Comment:
Why are you forcibly enabling the extension here, and the feature below? I'm not sure I understand the reason for this, but especially for this extension where there's no more granular enable than the device feature this could actively break or change behaviour in captures that don't use the extension by changing which viewports are used.